extensions/binding/org.eclipse.smarthome.binding.digitalstrom/ESH-INF/i18n/digitalstrom_en.properties

```diff
@@ -0,0 +1,127 @@
+# binding
+BINDING_NAME = digitalSTROM Binding
+BINDING_DESC = The digitalSTROM Binding integrates the digitalSTROM-System and allows to control the included devices and call scens.
+
+#dss-bridge
```

Comment:
please go through the correct spelling of this file yourself.
A few findings: 
- priorities, not priority's
- information, not informations
- don't use the word "Thing"
- shorten the labels

etc.